src/main/java/de/komoot/photon/PhotonDoc.java

```diff
@@ -81,8 +81,6 @@ public boolean isUsefulForIndex() {
 
 		if(name.isEmpty()) return false;
 
-		if(linkedPlaceId > 0) return false;
-
-		return true;
+		return linkedPlaceId <= 0;
```

Comment:
intellij is continuously recommending to simplify this code this way, but i think it is very hard to understand this way